### PR TITLE
gh-3 Rewrite README

### LINKED ISSUE #3: Update the README file with clearer instructions
The current README instructions are not very clear and not directed towards the customisation for the NHS Data Dictionary Mauro toolset. The whole README file could be rewritten to focus on the following:

- Requirements for installation
- Basic setup
    - How to build Docker images
    - How to run the Docker containers
 - Details of using the running containers e.g. 
     - Location of log files and volumes, 
     - Ports
     - Accessing the web apps
     - NHS specifics e.g. ingesting via API endpoint
     - etc

A lot of the original forked README contains a lot of reference material which is probably not relevant. This could be cleaned up too, focusing only on the details that are actually relevant for the NHS Data Dictionary maintenance.

### PR DESCRIPTION
**Pull request by pjmonks**
- Simplifies the setup and running instructions
- Adds references to the NHSD specific tools

Resolves #3 

**Note:** this is effectively a rewrite of the README file, so the git diff will show significant changes. My suggestion is to review the README file on its own and see if it is clear, understandable and contains all the relevant information (or suggest additional notes to add).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,471 +1,187 @@
...
-```yml
-services:
-  mauro-data-mapper:
-    environment:
-        database.host: another-host
+Once the images are ready, run the containers in detached mode as follows:
 
+```bash
+docker compose up -d
 ```
```
Comment (mhocc) [inline README.md:54]: It is possible to run `docker compose up -d` without needing a `docker compose build`.  I've not tried this but it should also work for updating.

File: README.md [hunk 2]
```diff
@@ -1,471 +1,187 @@
...
-### Cleaning up docker
+- Source (host): `shared_volumes/logs/maurodatamapper/mauro-data-mapper/mauro-data-mapper.log`
+- Destination (container): `/usr/local/tomcat/logs/mauro-data-mapper/mauro-data-mapper.log`
 
-Continually building docker images will leave a lot of loose snapshot images floating around, occasionally make use of:
+# Cleanup
+
+Continually building docker images will leave a lot of loose snapshot images floating around. Use the following commands to cleanup dangling resources: 
 
 * Clean up stopped containers - `docker rm $(docker ps -a -q)`
```
Comment (mhocc) [inline README.md:118]: This shell syntax will work in a bash prompt but not in a Windows command prompt

File: README.md [hunk 3]
```diff
@@ -1,471 +1,187 @@
...
-* Clean up untagged images - `docker rmi $(docker images | grep "^<none>" | awk "{print $3}")`
+* Clean up dangling images - `docker rmi $(docker images -q --filter "dangling=true")`
 * Clean up dangling volumes - `docker volume rm $(docker volume ls -qf dangling=true)`
```
Comment (mhocc) [inline README.md:121]: There is also a `docker system prune` that can also clean up dangling images stopped containers, unused networks etc.

**Review by mhocc — APPROVED** [review]
A huge improvement on the previous readme

File: README.md (hunk 2 above)
Comment (pjmonks) [reply README.md:118]: Strange, seems to work in PowerShell

File: README.md (hunk 1 above)
Comment (pjmonks) [reply README.md:54]: I've updated to include this detail, thanks 👍

File: README.md (hunk 3 above)
Comment (pjmonks) [reply README.md:121]: I've updated to include this, thanks 👍

**Pull request merged**